Makefile

```diff
@@ -28,12 +28,10 @@ dist/build/uBlock0.nodejs: tools/make-nodejs.sh $(sources) $(platform) $(assets)
 nodejs: dist/build/uBlock0.nodejs
 
 lint: nodejs
-	eslint -c platform/nodejs/eslintrc.json \
-		dist/build/uBlock0.nodejs/js \
-		dist/build/uBlock0.nodejs/*.js
+	cd dist/build/uBlock0.nodejs && npm install && npm run lint
```

Comment:
Now this is just like the `test` target below, and both run `npm install`. `npm pack` should ensure that nothing in the `node_modules` directory gets included in the archive.